lib/binding_web/test/language-test.js

```diff
@@ -0,0 +1,100 @@
+const {assert} = require('chai');
+let JavaScript;
+
+describe("Language", () => {
+
+  before(async () =>
+    ({JavaScript} = await require('./helper'))
+  );
+
+  describe(".fieldCount", () => {
+    it("returns a number", () => {
+        assert.equal(34, JavaScript.fieldCount);
+    });
+  });
```

Comment:
It looks like the indentation is inconsistent in some parts of this file. Could you adjust it to use 2 spaces throughout?